pkg/vere/main.c

```diff
@@ -1170,43 +1179,80 @@ _cw_eval_get_input(FILE* fil_u, size_t siz_i)
   return c3_realloc(str_c, len_i);
 }
 
+/* _cw_eval_get_newt(): read a newt-encoded jammed noun from file and return a
+**                      malloc'd byte buffer
+*/
+c3_y*
+_cw_eval_get_newt(FILE* fil_u, c3_d* len_d)
+{
+  //  TODO: can't reuse u3_newt_decode; coupled to reading from pipe
+  c3_d  i;
+  c3_y* byt_y = c3_realloc(NULL, 5); //  size is start size
```

Comment:
Why not use `c3_malloc()` to allocate for the first time instead of using the edge case behavior of `c3_realloc()`?

Reply:
`_cw_eval_get_newt` used `_cw_eval_get_string` as a base, so I wanted to keep their styles identical. I assumed that this is some sort of C trick with which I am unfamiliar that is faster or more defensive. If it's not, I can replace the call with `malloc`. Should I do so in both functions?

Reply:
`realloc(NULL, size)` is equivalent to `malloc(size)`. From the `realloc` man page:

> If ptr is NULL, then the call [to realloc] is equivalent to malloc(size), for all  values
       of size.

I'd change both `realloc(NULL, /*snip*/)` calls to `malloc(/*snip*/)` calls.